### PR TITLE
[stable-2.9] iptables: Reorder comment postition (#71496)

### PR DESCRIPTION
**Pull request by samdoran**
##### SUMMARY

Backport of #71496 for Ansible 2.9

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

`lib/ansible/modules/iptables.py`

### CODE REVIEW

File: test/units/modules/system/test_iptables.py [hunk 1]
```diff
@@ -828,3 +828,82 @@ def test_iprange(self):
             '--dst-range',
             '10.0.0.50-10.0.0.100'
         ])
+
+    def test_insert_rule_with_wait(self):
```
Comment (aminvakil) [inline test/units/modules/system/test_iptables.py:832]: This function is unnecessary, it has not been on PR https://github.com/ansible/ansible/pull/71496.

**Comment by aminvakil** [comment]
Could you please check this https://github.com/ansible/ansible/pull/72551 ?

**Comment by samdoran** [comment]
Yeah, I noticed I didn't merge this cleanly. Just didn't have time to clean it up yesterday.

**Comment by samdoran** [comment]
Superseeded by #72551. Thanks @aminvakil!

**Comment by aminvakil** [comment]
@samdoran 
I know you've probably a lot on your hand.
I'd be glad to create backport for my PR and help!

**Pull request closed without merging**